### ISSUE TITLE
audit-avt1-accordion:  multiple elements region role

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon-website#1551]
## Environment:

> Operating system: Mac
> Browser: Chrome 
> AVT :one:
> Page: [Components - Accordion: Accessibility](https://www.carbondesignsystem.com/components/accordion/accessibility/)
> This issue is affecting 4 total violations

## Detailed Description: 

Multiple elements with "region" role do not have unique labels

<img width="1680" alt="Screen Shot 2020-08-11 at 1 41 51 PM" src="https://user-images.githubusercontent.com/32720851/89930712-2a29f380-dbd9-11ea-81a8-368da174a40c.png">

## What to do: 
For each element with a "region" role, either set the value of the aria-labelledby attribute of each region to the id value of any visible

## Why is this important?
When there are multiple elements with a "region" role, they must have unique labels that clearly describe the region's content.
